modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_paragraph.scss

```diff
@@ -0,0 +1,128 @@
+@charset "UTF-8";
+
+.su-wysiwyg-text {
+
+  p {
+    @include big-paragraph;
+    @include modular-spacing('padding-bottom', 1);
+    @include margin(0 null null null);
+  }
+
+  // Caption Styles.
+  .caption {
+    figcaption {
+      text-align: center;
+    }
+  }
+
+  // Big bold beautiful callout text.
+  .su-callout-text {
```

Comment:
This selector probably needs the 
```
@at-root #{selector-unify(&, p)},
    & {
```
approach as this selector will lose out to the `p` selector on the font size.